src/repository/couch/CredentialRepository.ts

```diff
@@ -27,7 +27,7 @@ export class CredentialRepository implements CredentialInterface {
     }))
   }
   public async _insert(type_id: string | null, credential: any): Promise<{}> {
-    if (credential.origin === "me") {
+    if (credential.origin === "me" || credential.origin === "null") {
```

Comment:
**Please remove this.** `null !== "null"`, and we do not support a string containing the value `"null"` for this. Furthermore, both `type_id` and `origin` should be null here.